### DISCUSSION TITLE
Is it correct that the xplora_watch.see only update the watch tracker and not the other sensors

### DISCUSSION THREAD

**Discussion by smarthomefamilyverrips** [Ludy87/xplora_watch#167] [Q&A]
@Ludy87 

Is it correct that the xplora_watch.see only update the watch tracker and not the other sensors?

Is there a way to get more often updates for other sensors also?

**Comment by Ludy87** [comment]
which sensor should be updated separately, many of the sensors are linked.

**Reply by smarthomefamilyverrips** [reply]
At me only the tracker updates if I run the command

**Reply by Ludy87** [reply]
This only happens when a service of the integration is used regularly. See the annotation: https://github.com/Ludy87/xplora_watch#features

**Reply by smarthomefamilyverrips** [reply]
@Ludy87, I see, what you would suggest as a good time interval for the watch.see service call to set in my automation?

**Reply by Ludy87** [reply]
the balance is a dichotomy between battery and timeliness. 3600 seconds is enough for me.

**Comment by smarthomefamilyverrips** [comment]
@Ludy87 that is a much bigger interval as I use 😀 ..

Could I ask as I just have the watches: 

Does the tracker position update also more regular in case of position change without doing a watch.see service call?

**Reply by Ludy87** [reply]
No, I asked myself the same question. I should be able to read FCM from the app, but I don't think it's possible.

**Reply by smarthomefamilyverrips** [reply]
@Ludy87 OK thank you! And thank you for this great integration..... I did use Myki watch because did think could integrate that in HA, but after bought one watch I found out only works with older watches that use old webapp.... so now bought 2 X6Play watches and am very happy they do integrate because of you! 😊👍🏻
